### PR TITLE
doc: release: Document release quality metrics

### PR DESCRIPTION
**Pull request by dleach02**
Document the bug count metrics used to gate the release process

Signed-off-by: David Leach <david.leach@nxp.com>

### CODE REVIEW

File: doc/development_process/release_process.rst [hunk 1]
```diff
@@ -98,6 +99,25 @@ Here is the description of the various moderation levels:
   - Bug Fixes: P1 and P2
   - Documentation + Test Coverage
 
+Release Quality Metrics
+***********************
+
+The current backlog of prioritized bugs shall be used as a quality metric to
```
Comment (ioannisg) [inline doc/development_process/release_process.rst:105]: Better say "triaged bugs"?

File: doc/development_process/release_process.rst [hunk 2]
```diff
@@ -98,6 +99,25 @@ Here is the description of the various moderation levels:
   - Bug Fixes: P1 and P2
   - Documentation + Test Coverage
 
+Release Quality Metrics
```
Comment (nashif) [inline doc/development_process/release_process.rst:102]: Maybe 'Quality release criteria (QRC)'?

Comment (jukkar) [reply doc/development_process/release_process.rst:102]: > Quality release criteria (QRC)'

That sounds like we have a quality release, wouldn't it better to say "Release Quality Criteria", so a release has a quality criteria (and not the other way around). This is probably just semantics :)

Comment (dleach02) [reply doc/development_process/release_process.rst:102]: I have no strong preference here. I've been calling them quality metrics for a while and hence the language but I concede that we don't seem to have a common language here. But if we want to change my language I would lean towards @jukkar suggested phrasing.

File: doc/development_process/release_process.rst (hunk 1 above)
Comment (dleach02) [reply doc/development_process/release_process.rst:105]: I think prioritized is better. We do a triage action to get to a prioritization disposition of the bugs and it is the prioritization states that matter in this guide. Just want to reinforce that to the reader.

File: doc/development_process/release_process.rst (hunk 2 above)
Comment (dleach02) [reply doc/development_process/release_process.rst]: I've gone with @jukkar suggested wording

**Review by nashif — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]
LGTM, thank you!

**Review by ioannisg — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**